Samples/HelloWorld/OrleansClient/Program.cs

```diff
@@ -16,8 +16,27 @@ public class Program
         static int Main(string[] args)
         {
             var config = ClientConfiguration.LocalhostSilo();
-            const int initializeAttemptsBeforeFailing = 5;
-            int attemp = 0;
+            try
+            {
+                InitializeWithRetries(config, initializeAttemptsBeforeFailing: 5);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine($"Orleans client initialization failed failed due to {ex}");
+
+                Console.ReadLine();
+                return 1;
+            }
```

Comment:
if you are going to wrap the whole InitializeWithRetries code block into its own function, you should include the exception handling inside it too. 

Like I described before, the exception handling is part of InitializeWithRetries, whether you want to print it to a log file or a console. if people are curious about exception handling on InitializeWithRetries, dig one layer deeper.